configs/legacy/README.md

```diff
@@ -0,0 +1,46 @@
+# Legacy Configs in MMDetection V1.x
+
+Configs in this directory implements the legacy configs used by MMDetection V1.x and its model zoos.
+
+To help users convert their models from V1.x to MMDetection V2.0, we provide v1.x configs to inference the converted v1.x models.
+Due to the BC-breaking changes in MMDetection V2.0 from MMDetection V1.x, running inference with the same model weights in these two version will produce different results. The difference will cause within 1% AP absolute difference as can be found in the following table.
+
+## Usage
+
+To upgrade the model version, the users need to do the following steps.
+
+### 1. Convert model weights
+Since all the detector's classification is reordered, all the legacy model weights need to go through the conversion process.
+For two-stage detectors, their wegihts need to be upgraded since MMDetection V2.0 refactors all the two-stage detectors with `RoIHead`.
+For example, to convert Mask R-CNN and Faster R-CNN, their weights need to be converted as the following
+
+```bash
+python -u tools/upgrade_model_version.py ${OLD_MODEL_PATH} ${NEW_MODEL_PATH} --num-classes ${NUM_CLASSES} [--upgrade-retina] [--ssd] [--reg-cls-agnostic]
+
+```
+- OLD_MODEL_PATH: the path to load the model weights in 1.x version.
+- NEW_MODEL_PATH: the path to save the converted model weights in 2.0 version.
+- NUM_CLASSES: number of classes of the original model weights. Usually it is 81 for COCO dataset, 21 for VOC dataset.
+The number of classes in V2.0 models should equal to the number of classes in V1.x models + 1.
+
+Optional arguments:
+- `--upgrade-retina` Option for some RetinaNet models trained in older codebase (lower than V1.0), which will also upgrade the model weights' name.
+- `--ssd` Option for the SSD Detector, the SSD Detector only need to reorder its classification branch.
+- `--reg-cls-agnostic` Option for Cascade methods whose regression branch is class agnostic.
```

Comment:
Config is stored in the meta of checkpoint and we may obtain such information from the config.